### PR TITLE
Add docs polish for 0.5.3 release

### PR DESCRIPTION
**Pull request by nathanleclaire**
Some polish / version-type updates to our docs for the 0.5.3 release.

Will comment on what's changed inline.

cc @londoncalling @dgageot @jeanlaurent please take a look

Signed-off-by: Nathan LeClaire <nathan.leclaire@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,23 +9,29 @@ configures the Docker client to talk to them.
...
-Creating client certificate: /home/username/.docker/machine/certs/cert.pem
-Image cache does not exist, creating it at /home/username/.docker/machine/cache...
-No default boot2docker iso found locally, downloading the latest release...
-Downloading https://github.com/boot2docker/boot2docker/releases/download/v1.6.2/boot2docker.iso to /home/username/.docker/machine/cache/boot2docker.iso...
-Creating VirtualBox VM...
-Creating SSH key...
-Starting VirtualBox VM...
-Starting VM...
-To see how to connect Docker to this machine, run: docker-machine env dev
+$ docker-machine create -d virtualbox default
```
Comment (nathanleclaire) [inline README.md]: Updated this `create` output in several places to be more current and also to favor `default` over `dev`.

File: docs/get-started.md [hunk 2]
```diff
@@ -15,41 +15,48 @@ Let's take a look at using `docker-machine` for creating, using, and managing a
 Docker host inside of [VirtualBox](https://www.virtualbox.org/).
 
 First, ensure that [the latest
-VirtualBox](https://www.virtualbox.org/wiki/Downloads) is correctly installed on
-your system.
+VirtualBox](https://www.virtualbox.org/wiki/Downloads) is correctly installed
+on your system.
```
Comment (nathanleclaire) [inline docs/get-started.md]: Word wrap pedantism.

File: docs/get-started.md [hunk 3]
```diff
@@ -139,23 +146,3 @@ Make sure to specify the machine name as an argument:
 
     $ docker-machine stop dev
     $ docker-machine start dev
-
```
Comment (nathanleclaire) [inline docs/get-started.md]: Don't worry, this hasn't been removed: It's been moved to the landing page to ensure that it's presented clearly and at the top level.

File: docs/index.md [hunk 4]
```diff
@@ -31,23 +31,62 @@ Machine on Linux or for installing Machine alone without Docker Toolbox, see the
...
-
-To create a virtual machine, you supply Docker Machine with the name of the driver you want use. The driver represents the virtual environment. For example, on a local Linux, Mac, or Windows system the driver is typically Oracle Virtual Box. For cloud providers, Docker Machine supports drivers such as AWS, Microsoft Azure, Digital Ocean and many more. The Docker Machine reference includes a complete [list of the supported drivers](drivers/index.md).
-
-Since Docker runs on Linux, each VM that Docker Machine provisions relies on a base operating system. For convenience, there are default base operating systems. For the Oracle Virtual Box driver, this base operating system is the `boot2docker.iso`. For drivers used to connect to cloud providers, the base operating system is Ubuntu 12.04+. You can change this default when you create a machine. The Docker Machine reference includes a complete [list of the supported operating systems](drivers/os-base.md).
+Docker Machine allows you to provision Docker on virtual machines that reside
```
Comment (nathanleclaire) [inline docs/index.md]: This is also just wrap pedantism.  No content has changed.

File: docs/index.md [hunk 5]
```diff
@@ -31,23 +31,62 @@ Machine on Linux or for installing Machine alone without Docker Toolbox, see the
...
+OS, ARCH, the path to your current shell and, the history of the last command as
+you could see it with a `--debug` option.  This data is sent to help us pinpoint
+recurring issues with `docker-machine` and will only be transmitted in the case
+of a crash of `docker-machine`.
+
+If you wish to opt out of error reporting, you can create a `no-error-report`
+file in your `$HOME/.docker/machine` directory, and Docker Machine will disable
+this behavior.  e.g.: 
+
+    $ mkdir -p ~/.docker/machine && touch ~/.docker/machine/no-error-report
```
Comment (nathanleclaire) [inline docs/index.md]: I did add this bit, just so users could easily copy-paste the opt-out command if desired.

**Comment by dgageot** [comment]
LGTM

**Pull request merged**